server_load_profile.py

```diff
@@ -6,44 +6,53 @@
 
 app = Flask(__name__)
 
+# Constants
+DATE_FORMAT = '%Y-%m-%d'
+EXPECTED_ARRAY_SHAPE = (2, 24)
+FILEPATH = r'.\load_profiles.npz'
 
+def get_load_forecast(year_energy):
+    """Initialize LoadForecast with the given year_energy."""
+    return cl.LoadForecast(filepath=FILEPATH, year_energy=float(year_energy))
```

Comment:
Should `FILEPATH` handled by `pathlib` to create a absolute path instead of relative one if work dir changed?

Reply:
Gerne, wobei die Datei fix ist.